packages/app/src/cli/services/function/build.test.ts

```diff
@@ -155,7 +155,7 @@ describe('runJavy', () => {
 })
 
 describe('runFunctionRunner', () => {
-  test('calls function runner to execute function locally', async () => {
+  test('calls function runner to execute function locally', {timeout: 20000}, async () => {
```

Comment:
I also believe these should be stubbed too. In [this PR](https://github.com/Shopify/cli/pull/4404/files#diff-674e37c8b6c1dd31f06e0f74865766428e378534290b123fdd1ff6fb491a5090R13), I've refactored the `runFunction` function and stubbed the install in the test.